src/Microsoft.Extensions.ObjectPool/Microsoft.Extensions.ObjectPool.csproj

```diff
@@ -8,4 +8,8 @@
     <PackageTags>pooling</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
```

Comment:
I'm not sure we want this type of dependency in the object pool.

Reply:
OK. So in this case take the variant with the abstract base class?

Reply:
The variant with the abstract base class -> https://github.com/aspnet/Common/pull/318